app/components/mobile-dialog.js

```diff
@@ -0,0 +1,66 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+export default class MobileDialogComponent extends Component {
+  @tracked toShow = this.checkDeviceType();
+
+  @action
+  closeDialog() {
+    this.toShow = false;
+  }
+
+  @action
+  openRDSApp() {
+    this.openApp();
+  }
+
+  checkDeviceType() {
+    let regexp = /android|iphone|kindle|ipad/i;
+    let details = navigator.userAgent;
+    let isMobileDevice = regexp.test(details);
+  
+    if (isMobileDevice) {
+      console.log('true');
+      return true;
+    } else {
+      console.log('false');
+      return false;
+    }
+  };
+
+  openApp() {
+    var flag = false;
+    var appScheme = 'app://realdevsquad.com';
+    var fallbackURL =
+      'https://play.google.com/store/apps/details?id=com.github.android'; // For demo. It will replace with app playstore url
+  
```

Comment:
Fix this below error
<img width="421" alt="image" src="https://github.com/Real-Dev-Squad/website-my/assets/107163260/8d21e5f2-54a2-4029-83c9-971909b42a62">